docs/data/base/components/click-away-listener/click-away-listener-pt.md

```diff
@@ -1,9 +1,8 @@
 ---
-product: material-ui
+product: base
```

Comment:
some pages like this one had outdated `product` attribute

Reply:
Looks like these are all in the translated pages, which we don't generally edit directly. (I'm not sure why exactly.)

Reply:
> I'm not sure why exactly.

Tries to answer this question: https://www.notion.so/mui-org/Translations-1c9ad92060cb4bb49669ab5e1c0bbb8e.

Reply:
So it looks like Crowdin will update `product` field as well, but only when there's an update to this page's translation.
If I understand it correctly, there should be no problem with updating it manually in this PR.